book/chapters/jets.ipynb

```diff
@@ -23,137 +23,102 @@
    "source": [
     "The first thing to do is import what we need in our environment. This will be the case at the top of every chapter, but we show it for completness.\n",
     "\n",
-    "* The datasets. See the [DataSet](ch-data-samples)  chapter for more information.\n",
+    "* Function from config file that returns dataset dictionary\n",
     "* Various utilities for plotting and array manipulation\n",
     "* Some helpers for accessing attributes below (`cpp_float, cpp_vfloat`)"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from config import get_dataset\n",
+    "from func_adl_servicex_xaodr22 import FuncADLQueryPHYSLITE, cpp_float, cpp_vfloat\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import uproot\n",
+    "import awkward as ak"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To show how to use func_adl with xaod on the jet collection we will get $ p_{T} $ data above 30GeV. To do this we must build the query using func_adl and pass it to the get_dataset function. To do this we utilize func_adl Where() function and then select the resulting data using Select() where we also scale the data to GeV. We also select $ \\eta $ and emf data for later in the tutorial."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
    "metadata": {},
    "outputs": [
     {
-     "name": "stderr",
+     "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Found backend type matching \"xaod\". Matching by type is depreciated. Please switch to using the \"name\" keyword in your servicex.yaml file.\n",
-      "Found backend type matching \"xaod\". Matching by type is depreciated. Please switch to using the \"name\" keyword in your servicex.yaml file.\n",
-      "WARNING:root:Fetched the default calibration configuration for a query. It should have been intentionally configured - using configuration for data format {default_calibration_name()}\n"
+      "\u001b[2K5l\n",
+      "\u001b[?25h"
      ]
     }
    ],
    "source": [
-    "# from config import ds_zee_r21 as ds\n",
-    "# from config import ds_jz2_exot15_r21\n",
-    "from config import ds_ttbar_r22 as ds\n",
-    "import matplotlib.pyplot as plt\n",
-    "import awkward as ak\n",
-    "import numpy as np"
+    "query = FuncADLQueryPHYSLITE()\n",
+    "jets_per_event = query.Select(lambda e: e.Jets('AnalysisJets'))\n",
+    "jet_info_per_event = jets_per_event.Select(\n",
+    "    lambda jets: {\n",
+    "        'pt': jets.Where(lambda j: (j.pt() / 1000 > 30)).Select(lambda j: j.pt() / 1000),\n",
+    "        'eta': jets.Select(lambda j: j.eta()),\n",
+    "        'emf': jets.Select(lambda j: j.getAttribute[cpp_float]('EMFrac'))\n",
+    "        }\n",
+    ")\n",
+    "\n",
+    "ds = get_dataset(jet_info_per_event)"
```

Comment:
ds usually means dataset - the source. Rather than the resulting output. So actually, we shouldn't call it `get_dataset`.